public/hackathon/index.html

```
@@ -0,0 +1,1844 @@
+              match AgentKit → Connections exactly.
+            </p>
+            <pre><code>SCALEKIT_CLIENT_ID=skc_...
+SCALEKIT_CLIENT_SECRET=...
+SCALEKIT_ENV_URL=https://yourorg.scalekit.com
+GMAIL_CONNECTION_NAME=your-exact-connection-name</code></pre>
+          </div>
+
+          <div class="tabs" data-tabs style="margin-top: 1.5rem">
+            <div class="tablist" role="tablist" aria-label="SDK language">
+              <button
+                class="tab"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)